tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml

```diff
@@ -25,7 +25,7 @@ tests:
       - nrf5340dk/nrf5340/cpuapp
       - nrf5340dk/nrf5340/cpuapp/ns
       - nrf54l15pdk/nrf54l15/cpuapp
-    tags: sysbuild
+    tags: sysbuild bluetooth
```

Comment:
I think we should use `crypto` for Fast Pair crypto tests (these tests actually do not use Bluetooth). @kapi-no what do you think?

Reply:
For storage tests we could stick to `bluetooth` tag (Aleksander Strzeboński will add BLE bond management soon, so we may take it as "good enough" approach)

Reply:
ok

Reply:
Discussed offline with @nordic-piks. We will stick to `bluetooth` tag both Fast Pair crypto and storage tests. Moving to `crypto` tag would require to add our tests' path under the `crypto` tag (otherwise our Fast Pair crypto tests would not run if a PR modifies both the test and any other file).

Reply:
But will it be run if someone modifies crypto backend?

Reply:
yes, deps are added:
     - modules/crypto/mbedtls/
    - modules/crypto/oberon-psa-crypto/
    - modules/crypto/tinycrypt/
    - nrf/subsys/nrf_security/
    - nrfxlib/crypto/
    
    https://github.com/nrfconnect/sdk-nrf/blob/0217530428c8552cff22a91cf4059e5cbf630764/scripts/ci/tags.yaml#L66-L70

Reply:
Oh, I didn't understand it at first. So we add a tag to **not** run a test when the tag-related thing changes? E.g. if we have `bluetooth` tag in testcase and something related to Bluetooth changes then this testcase won't be run. @nordic-piks, am I right?

Reply:
Opposite, We do not run tests marked with tag when dependencies listed for that tag are not changed (files changed in PR)

Reply:
Okay, so we have `bluetooth` tag and `bluetooth` depends on crypto, settings etc. Now that makes sense. Thanks!